ehcache-impl/src/main/java/org/ehcache/impl/persistence/DefaultLocalPersistenceService.java

```diff
@@ -115,6 +140,19 @@ private void internalStart() {
   @Override
   public synchronized void stop() {
     if (started) {
+      try {
+        if (cleanFile.createNewFile()) {
+          LOGGER.debug("clean file is created.");
+        } else {
+          LOGGER.warn("clean file still exists on shutdown. This may be due to a filesystem consistency issue, or concurrent modification of the filesystem by another thread and/or process. Be aware for the potential of undetected storage corruption causing cache failures and/or data consistency issues." +
+            "\n Hint: clean file exists on service start-up, indicates service was stopped cleanly last time. It gets created while the service is stopped and it should be deleted while the service is started." +
+            "\n Action: Please verify there permission to delete the file and delete the root directory prior to start the service again.");
+        }
+      } catch (IOException e) {
+        LOGGER.warn("clean file could not be created on shutdown. This will cause the persisted state to be considered unclean on the next service start, which may result in unplanned data loss. Reason: " + e.getMessage() +
+          "\n Hint: clean file exists on service start-up, indicates service was stopped cleanly last time. It gets created while the service is stopped and it should be deleted while the service is started." +
```

Comment:
Again, I think the additional lines are not needed. The first line states what has happened, and what may result, and again please capitalize the 'c'.